examples/analytics-iframe-transport-remote-frame.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Requests Frame</title>
+    <script>
+      /**
+       * To receive analytics events in a third-party frame, you must
+       * implement this method, with this signature. Within that method, you
+       * must create a function which processes the analytics events, and
+       * pass that function to the registerAmpAnalytics3pEventsListener() method
+       * of the object which is passed as a parameter to
+       * onNewAmpAnalyticsInstance().
+       * @param ampAnalytics  Call registerAmpAnalyticsEventListener() on this,
+       * passing your function which will receive the analytics events.
+       */
+      window.onNewAmpAnalyticsInstance = ampAnalytics => {
+        ampAnalytics.registerCreativeEventListener(event => {
+          // Now, do something meaningful with the AMP Analytics event
+          console.log("The page at: " + window.location.href +
+            " has received an event: " + event +
+            " from the creative with transport ID: " +
+            ampAnalytics.getTransportId());
```

Comment:
Simplification is done. Did not end up changing from "document.head" to "document.body" since the remote frame has no content and thus no body.